### PR TITLE
Spc 729 cosine distance vs euclidian distance

### PR DESCRIPTION
**Pull request by Jarelk**
Om pylint errors te vermijden (pylint errors die somehow in develop zijn gekomen, eigenlijk) heb ik functies van fast-reid en torchreid in de "interface" reidentifier geplaatst. Oftewel de interface is geen interface meer.

### CODE REVIEW

File: CameraProcessor/processor/pipeline/reidentification/i_re_identifier.py [hunk 1]
```diff
@@ -27,18 +28,6 @@ def feature_map_size(self):
         """
         raise NotImplementedError("Feature map size getter not implemented")
 
-    def extract_features(self, frame_obj, bbox):
```
Comment (Sxord) [inline CameraProcessor/processor/pipeline/reidentification/i_re_identifier.py:30]: This function should stay in the reidentifier interface

File: CameraProcessor/processor/pipeline/reidentification/pytorch_re_identifier.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        Args:
+            frame_obj (FrameObj): frame object storing OpenCV frame and timestamp.
+            bbox (BoundingBox): BoundingBox object that stores the bounding box from which we want to extract features.
+
+        Returns:
+            [float]: Feature vector of a single bounding box.
+        """
+        raise NotImplementedError("Extract features function not implemented")
+
+    def extract_features_from_cutout(self, cutout):
```
Comment (Sxord) [inline CameraProcessor/processor/pipeline/reidentification/pytorch_re_identifier.py:74]: already contained in the interface, no need to override the function if nothing is changed (still throws same error)

**Review by Sxord — CHANGES_REQUESTED** [review]

File: CameraProcessor/processor/pipeline/reidentification/pytorch_re_identifier.py (hunk 2 above)
Comment (Jarelk) [reply CameraProcessor/processor/pipeline/reidentification/pytorch_re_identifier.py:54]: You are right, moved it

File: CameraProcessor/processor/pipeline/reidentification/i_re_identifier.py (hunk 1 above)
Comment (Jarelk) [reply CameraProcessor/processor/pipeline/reidentification/i_re_identifier.py]: Thou art correct, I hath relocated the code part in question to the IRe-identifier

**Review by Sxord — APPROVED** [review]

**Review by saberzero1 — APPROVED** [review]

**Pull request merged**